.huskyrc.js

```diff
@@ -0,0 +1,14 @@
+const { config } = require('@dhis2/cli-style')
+const husky = require(config.husky)
+
+const tasks = arr => arr.join(' && ')
+
+module.exports = {
+    hooks: {
+        ...husky.hooks,
+        'pre-commit': tasks([
+            'd2-style js check --staged',
```

Comment:
Should we destructure `...husky.hooks['pre-commit']` here as well? I don't know if it's the intention that we overwrite `pre-commit` or merge with the ones supplied by the dhis2 config.

Reply:
This file was generated by running the command `npx d2-style install project/react --force`. So if you would like a change here, you should go via that repo.

More info:
- [This](https://github.com/dhis2/cli-style/blob/master/config/husky-frontend.local.js) is where the file comes from 
- All files in this commit 4211085abc6db401ed6a1a2a14a2145986e95d71 were auto-generated, apart from the change to `.browserslistrc`

Reply:
Ah yeah of course. In that case I assume this is fine 👍